seed.js

```diff
@@ -5,6 +5,16 @@
 
 // var new_campsite = {description: "Sharp rocks. Middle of nowhere."}
 
+//var info_list =
+
```

Comment:
need hard cord to test the database if it is working.

Reply:
+1